src/utils/monitoring-sites-data.js

```diff
@@ -111,6 +130,46 @@ const setRiverFlowSitesInBoundingBox = async (map, corners) => {
   map.getSource("riverFlowSites").setData(sites);
 };
 
+const setBristolWaterQualitySitesInBoundingBox = async (map) => {
+  const query = bristolWaterQualityQuery(map);
+  const response = await fetch(query).then((response) => response.json());
+  const records = response.records;
+  const siteIds = response.facet_groups[0].facets.map((facet) => facet.name);
```

Comment:
Are we certain the response always contains `facet_groups`? And that that is always an array with at least one item? This will throw an exception if so.. maybe to be safe you could make the chaining [optional](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining) (just turn the `.`s into `?.`s), so that in case any link is missing `siteIds` will just end up `null`.